plenum/server/node.py

```diff
@@ -336,6 +336,7 @@ def __init__(self,
         # of `1:101`
         self.txn_seq_range_to_3phase_key = {}  # type: Dict[int, IntervalTree]
         self._view_change_in_progress = False
+        self.propagate_primary = False
```

Comment:
The quorum of `ViewChangeDone` msgs is different depending on whether we're doing a real view change, or just propagating viewNo and Primary from `CurrentState` messages sent to a newly joined Node.

Added a comment.